src/main/java/Main.java

```diff
@@ -38,8 +39,9 @@ public static void main(String[] args) {
         CourseDbGateway courseDbGateway = new CourseDbGateway(dbConnection);
 
         // CreateStudent Use Case
+        StudentFactory studentFactory = new StudentFactory();
         CreateStudentPresenter createStudentPresenter = new CreateStudentPresenter();
-        CreateStudentInteractor createStudentInteractor = new CreateStudentInteractor(studentDbGateway, createStudentPresenter);
+        CreateStudentInteractor createStudentInteractor = new CreateStudentInteractor(studentDbGateway, createStudentPresenter, courseDbGateway, studentFactory);
```

Comment:
This change reflects the fact that new users must have all courses added to the database with respective null grades for each course. I was wondering why we have added this but I understand we require it because we changed how our database is strucuted.